### DISCUSSION TITLE
Immersive breath play (Knockout lockdown)

### DISCUSSION THREAD

**Discussion by jimmyG1** [littlesera/LSCG#199] [Ideas]
win fully Koed and you have Immersive on it does the same as the ethers for a time YOU SET can be 1-10 minutes (a command can intly wack you up)

**Comment by littlesera** [comment]
To clarify, you mean to force a configurable "sleep" state on breathplay passout when immersive is enabled?

**Comment by jimmyG1** [comment]
yes

**Comment by jimmyG1** [comment]
so have a idea on win or if this will be a thang

**Comment by littlesera** [comment]
Implemented in #239
